### ISSUE TITLE
labels argument not working in plot.outbreaker_chains

### ISSUE THREAD

**Issue by pcdjohnson** [reconhub/outbreaker2#75]
The label argument in plot.outbreaker_chains isn't working for some plot types. When labels are supplied the labels on the y axis are still the default of 1:N. I'm using version 1.1.1 (currently both github and CRAN versions are 1.1.1). E.g.
```
plot(res, type = "t_inf", burnin = 2000, labels = names(dates))
plot(res, type = "kappa", burnin = 2000, labels = names(dates))
plot(res, type = "network", burnin = 2000, min_support = 0.01, labels = names(dates))

```
names(dates) is a character vector of the same length as dates and dna.

**Comment by finlaycampbell** [comment]
I will have a look once I have a spare moment, I know this feature isn't implemented yet for type network (though it shouldn't take long to do), perhaps there is a bug for the other two - I'll get back to you!

**Comment by finlaycampbell** [comment]
This is now fixed as of 283e978fa582df2861f967e98248a304c88b6c26!
